script.js

```diff
@@ -151,6 +154,33 @@ function loadAssets (hash, callback) {
     })
 }
 
+function KeyPressed() {
+    const keyName = event.key;
+    switch (event.key) {
```

Comment:
It's strange to use here event.key when one line before you declared variable with the same value.